src/components/messageInputContainer.tsx

```
@@ -61,123 +89,112 @@
+      if (silenceCheckInterval.current) {
+        clearInterval(silenceCheckInterval.current)
       }
-      isListeningRef.current = true
-      setIsListening(true)
 
-      if (realtimeAPIMode) {
-        audioChunksRef.current = [] // 音声チャンクをリセット
+      // 音声検出時刻を記録
+      lastSpeechTimestamp.current = Date.now()
+      speechEndedRef.current = false
+      console.log('🎤 無音検出を開始しました。無音検出タイムアウトの設定値に基づいて自動送信します。')
```

ESLint

[error] Replace `'🎤·無音検出を開始しました。無音検出タイムアウトの設定値に基づいて自動送信します。'` with `⏎········'🎤·無音検出を開始しました。無音検出タイムアウトの設定値に基づいて自動送信します。'⏎······`

(prettier/prettier)